Tests/LibPDF/TestPDF.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021, Simon Woertz <simon@woertz.at>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/MappedFile.h>
+#include <LibPDF/Document.h>
+#include <LibTest/Macros.h>
+#include <LibTest/TestCase.h>
+
+TEST_CASE(linearized_pdf)
+{
+    auto file = Core::MappedFile::map("/home/anon/Documents/pdf/linearized.pdf").release_value();
```

Comment:
For every commit that was added to this PR all tests are should be passing (both lagom and serenity/target). Since adding support for `lagom_test` was added in a "later" commit I currently cannot see how lagom would "pointlessly fail".